pyanaconda/bootloader.py

```diff
@@ -2303,16 +2418,27 @@ def install(self, args=None):
 
 
 # every platform that wants a bootloader needs to be in this dict
-bootloader_by_platform = {platform.X86: GRUB2,
-                          platform.EFI: EFIGRUB,
-                          platform.MacEFI: MacEFIGRUB,
-                          platform.PPC: GRUB2,
-                          platform.IPSeriesPPC: IPSeriesGRUB2,
-                          platform.NewWorldPPC: MacYaboot,
-                          platform.S390: ZIPL,
-                          platform.Aarch64EFI: Aarch64EFIGRUB,
-                          platform.ARM: EXTLINUX,
-                          platform.omapARM: EXTLINUX}
+if flags.cmdline.get("inst.legacygrub") == "1":
```

Comment:
looks like flags.cmdline is actually command line args to anaconda (anaconda.py) and not /proc/cmdline so this doesn't work right now.